fmriprep/__init__.py

```diff
@@ -44,5 +44,19 @@ def _new_version():
     iflogger.setLevel(level)
     if v is None:
         iflogger.warn('afni_vcheck executable not found')
-    return v
 Info.version = staticmethod(_new_version)
+
+from .due import due, Url
```

Comment:
Very sorry @effigies for the rushed up and sloppy work I have done!  I am usually better ;)  Fixed up commit is coming shortly!